test/extended/networking/livemigration.go

```diff
@@ -68,6 +70,15 @@ var _ = Describe("[sig-network][OCPFeatureGate:PersistentIPsForVirtualization][F
 
 					DescribeTable("[Suite:openshift/network/virtualization] should keep ip", func(netConfig networkAttachmentConfigParams, vmResource string, opCmd func(cli *kubevirt.Client, vmNamespace, vmName string)) {
 						var err error
+						l := map[string]string{
+							"e2e-framework": f.BaseName,
+						}
+						if netConfig.role == "primary" {
+							l[RequiredUDNNamespaceLabel] = ""
+						}
+						ns, err := f.CreateNamespace(context.TODO(), f.BaseName, l)
```

Comment:
https://github.com/openshift/origin/blob/478335660e225b02f1f80aead7f50a2f85bad075/test/extended/util/client.go#L426 this does a bunch more than just creating the namespace, I am wondering if the current approach could cause significant delays in pod creation for our tests. I am ok to keep it as is and see if there are no issues in the presubmits.